packages/terra-i18n/src/i18nSupportedLocales.js

```diff
@@ -1 +1 @@
-module.exports = ['ar', 'en', 'en-US', 'en-GB', 'es', 'es-US', 'es-ES', 'de', 'fi-FI', 'fr', 'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE'];
+module.exports = ['en', 'en-US', 'en-GB', 'en-AU', 'es', 'es-US', 'es-ES', 'de', 'fi-FI', 'fr', 'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE'];
```

Comment:
Do we need this file now? I know we used it to aggregate translations, but do we use it directly in the i18n Component?

Reply:
I debated with myself what to do with this. It is still used directly in the i18n component and it's related tests. With the code changes on the PR so far, this module is only used in the following files:
* Used to throw a warning the locale is not supported - https://github.com/cerner/terra-core/blob/728a5106f1e3fb113b3956cefeca996468b0ffb8/packages/terra-i18n/src/i18nLoader.js
* Used to test fallback behavior of unsupported locales - https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/src/terra-dev-site/test/i18n/DefaultI18n.test.jsx
* Used to test fallback behavior along with supported locale behavior -https://github.com/cerner/terra-core/blob/74e0af611b806303864132a53546ae054643ee3c/packages/terra-i18n/tests/wdio/i18n-spec.js

I feel like this module is kinda out of sight, out of mind, I'd prefer to have a singular place for our list of supported locales. I've kept this file for now but I'm open to ideas on how to handle it.

Reply:
I wonder if we need to worry about calling out that we are loading non-supported locales form terra-i18n anymore. We will log messages during aggregate-translations if the locale is not supported by terra but we will both aggregate and load whatever the user has defined.

since we do provided `ltr` and `rtl` styling for our components, I don't necessarily think it matters at this level if we are loading site assets we don't support. I am curious on what others think about this.

I do think it would be nice to drop these test fixtures: https://github.com/cerner/terra-core/tree/update-base-and-i18n-packages/packages/terra-i18n/tests/jest/fixtures. Our consumers ended up with these translations files aggregated when they run aggregate-translations and it logs warning messages for content we are shipping with the component.

Reply:
That's a good point about no longer needing to call out loading non-supported locales from terra-i18n. I'm fine dropping [the check from i18nLoader.js](https://github.com/cerner/terra-core/blob/728a5106f1e3fb113b3956cefeca996468b0ffb8/packages/terra-i18n/src/i18nLoader.js)

I'm on the fence about dropping the test fixtures. I see value in them but I'm also curious what others think as well.

Reply:
I tested removing the `fixtures` directory and all our Jest tests still pass. Now I'm not sure what they were used for, I assumed they were used for the Jest tests. It does seem like the `fixtures` directory is no longer needed. @emilyrohrbough do you recall what it was used for?

Reply:
Talked with @emilyrohrbough offline and decided the `fixtures` were used for aggregate-translation jest tests which have been removed in this branch so they are no longer needed. 

I've removed the `fixtures` directory in this commit: https://github.com/cerner/terra-core/pull/2288/commits/5a752387b5a83132ceaec80fe541b0909377d56f
I've removed the i18nSupportedLocales import and logic from `i18nLoaders.js` in this commit: https://github.com/cerner/terra-core/pull/2288/commits/aa792964b721b27641fe881781c8135f98ca7a13

There are 2 remaining places that the i18nSupportedLocales.js module is used in the terra-i18n package. 

* Used to test fallback behavior of unsupported locales - https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/src/terra-dev-site/test/i18n/DefaultI18n.test.jsx
* Used to test fallback behavior along with supported locale behavior -https://github.com/cerner/terra-core/blob/74e0af611b806303864132a53546ae054643ee3c/packages/terra-i18n/tests/wdio/i18n-spec.js

I'm open to idea on how to remove the i18nSupportedLocales.js module from those files.

Reply:
Talked with @mjhenkes about moving `i18nSupportedLocales.js` into the test directory to show that it is only needed for tests.

Reply:
Talked with @mjhenkes about just pulling this from toolkit.

Reply:
Removed this file and replaced it with an import from terra-toolkit in the 2 test files that still need the list of supported locales in this commit: https://github.com/cerner/terra-core/pull/2288/commits/954f2514715cd9e1d180a87da6c6a528435e9974